third_party/maya/lib/usdMaya/usdWriteJob.h

```diff
@@ -31,15 +31,18 @@
 #include "usdMaya/Chaser.h"
 
 #include "usdMaya/util.h"
-#include "usdMaya/MayaPrimWriter.h"
 #include "usdMaya/ModelKindWriter.h"
 
+#include "usdMaya/usdWriteJobCtx.h"
+
+#include <maya/MObjectHandle.h>
+
 #include <string>
 
 PXR_NAMESPACE_OPEN_SCOPE
 
 
-class usdWriteJob
+class usdWriteJob : public usdWriteJobCtx
```

Comment:
Are we expecting to derive more things from usdWriteJobCtx? If not, I think we'd prefer usdWriteJob to have a usdWriteJobCtx as a member instead and add a public accessor like:

 ```const usdWriteJobCtx& usdWriteJob::getContext() const``` 

or some such. From what we could tell, it seemed like that wouldn't be too big of a change to your code. What do you think?

Also, if we could add some simple class documentation to the top of usdWriteJobCtx that describes its purpose (e.g., serves as interface for export plugins) that'd be great.